_translator-files/po/de/Client-Troubleshooting.po

```diff
@@ -25,7 +25,7 @@ msgstr "de"
 #: ../wiki/en/Client-Troubleshooting.md:1
 #, no-wrap
 msgid "wiki"
-msgstr "wiki"
+msgstr "Wiki"
```

Comment:
This shouldn't be capitalised. I left a comment on Weblate but not sure if the translator has seen it. Correcting it in one string will correct all the other ones.

Reply:
Done